src/ACEtk/interpretation/EL03/MottScatteringCorrection.hpp

```diff
@@ -0,0 +1,42 @@
+class MottScatteringCorrection {
+  const Table& table;
+
+  auto h() const {
```

Comment:
Functions that are more than one logical line long (i.e., more than one semi-colon) need to be in their own file in order to match the style guide.